examples/webgl_shadow_contact.html

```diff
@@ -115,10 +115,11 @@
 					map: renderTarget.texture,
 					opacity: state.shadow.opacity,
 					transparent: true,
+					depthWrite: false,
 				} );
 				plane = new THREE.Mesh( planeGeometry, planeMaterial );
-				plane.material.polygonOffset = true;
-				plane.material.polygonOffsetFactor = -.1;
+				// make sure it's rendered ahead the fillPlane
```

Comment:
`renderOrder = 1` will actually make sure it's rendered after the fill plane

Reply:
right